src/assets/ReasonAsset.js

```diff
@@ -0,0 +1,28 @@
+const bsb = require('bsb-js');
+const fs = require('fs');
+const JSAsset = require('./JSAsset');
+const promisify = require('../utils/promisify');
+const readFile = promisify(fs.readFile);
+
+class ReasonAsset extends JSAsset {
+  async parse(code) {
+    // This runs BuckleScript - the Reason to JS compiler.
+    // Other Asset types use `localRequire` but the `bsb-js` package already
+    // does that internally. This should also take care of error handling in
+    // the Reason compilation process.
+    if (process.env.NODE_ENV !== 'test') {
```

Comment:
why not building in the tests?

Reply:
Building during test would require adding BuckleScript as a dev dependency. BS is a big native binary that compiles itself on install, which takes about 7 minutes.

The wrapper around BuckleScript, bsb-js, is pretty well tested, so the tests for Reason only test the file-loading logic.

Reply:
i see, ok.